<?xml version="1.0" encoding="UTF-8"?>
<launch>
    <rosparam file="$(find urquhart)/run_configs/geoSLAM/std.yaml" command="load" />

    <!-- Simulate robot observations -->
    <node pkg="urquhart" type="ultimate_path" name="sim_path" output="screen" />

    <!-- Local Bundle Adjustment Node -->
    <node pkg="urquhart" type="KeyframeGen" name="keyframe_maker" output="screen" />

    <!-- Global GraphSLAM Node -->
    <node pkg="urquhart" type="PolygonSLAM" name="graph_builder" output="screen" />
</launch>